### ISSUE TITLE
InvalidCharacterError when adding multiple classes on single condition in ng-class

### ISSUE THREAD

**Issue by rapropos** [angular/angular#5205]
An InvalidCharacterError is thrown on constructs like the following (I believe due to the space character):

```
<i [ng-class]="{'fa-spinner fa-spin': isWorking}"></i>
```

Adding multiple classes on a single condition like this worked for me in 1.4. A workaround is to separate the two classes:

```
<i [ng-class]="{'fa-spinner': isWorking, 'fa-spin': isWorking}"></i>
```

**Comment by estsauver** [comment]
This looks like it can be closed.

**Comment by pkozlowski-opensource** [comment]
Indeed, this was fixed by 4e02e003734d46f88b1e2cc137be0da7af70b328. Thnx @estsauver !
